framework/src/modules/chain/submodules/blocks/chain.js

```diff
@@ -432,40 +423,7 @@ class Chain {
  * @returns {Object} cb.err - Error if occurred
  */
 __private.afterSave = async function(block, cb) {
-	if (components && components.cache && components.cache.isReady()) {
-		library.logger.debug(
-			['Cache - chain afterSave', '| Status:', components.cache.isReady()].join(
-				' '
-			)
-		);
-		const delegateTransaction = block.transactions.find(
-			transaction =>
-				!!transaction && transaction.type === TRANSACTION_TYPES.DELEGATE
-		);
-		if (delegateTransaction) {
-			try {
-				await components.cache.removeByPattern(CACHE_KEYS_DELEGATES);
-				library.logger.debug(
-					[
-						'Cache - Keys with pattern:',
-						CACHE_KEYS_DELEGATES,
-						'cleared from cache on delegate transaction',
-					].join(' ')
-				);
-				await components.cache.deleteJsonForKey(CACHE_KEYS_TRANSACTION_COUNT);
-				components.cache.logger.debug(
-					`Cache - Keys ${CACHE_KEYS_TRANSACTION_COUNT} cleared from cache on chain afterSave`
-				);
-			} catch (err) {
-				library.logger.error(
-					['Cache - Error clearing keys', 'on chain afterSave function'].join(
-						' '
-					)
-				);
-			}
-		}
-	}
-
+	library.channel.publish('chain:confirmed_transactions:update', block);
```

Comment:
maybe better event name or payload?
it seems like it should send the transaction but it's sending block...

Reply:
It needs all the transactions; I will update it to send only the transactions instead of the rest of the block.

Reply:
Done! Please review again.